data_juicer/__init__.py

```diff
@@ -1 +1,58 @@
 __version__ = '0.1.3'
+
+import os
+
+import multiprocess as mp
+from loguru import logger
+
+__USE_CUDA = False
+__CUDA_COUNT = 0
+
+
+def use_cuda():
+    return __USE_CUDA
+
+
+def cuda_device_count():
+    return __CUDA_COUNT
+
+
+def setup_mp():
+    method = os.getenv('MP_START_METHOD', 'auto').lower()
+    if method == 'auto':
+        import torch
```

Comment:
If so, dependency `torch` might need to be moved to minimal_requires?

Reply:
I have replaced the torch method with a custom function, how about that?